router/routes/permission.js

```diff
@@ -3,19 +3,26 @@
  *     https://www.npmjs.com/package/acl
  */
 
```

Comment:
The scope of permissions have to be extended more,
Depending on the role column you've added to the CourseOfferin table,
Based on that a user might have to be given individual access to a certain courseoffering etc.

Lets discuss this in detail

Reply:
To check if user has a permission to a certain course, let's handle it through ACL. To filter out course with certain permission level requirements, use the attribute stored in the database.